deploy/docs/fluentd_otc_comparison.md

```diff
@@ -0,0 +1,208 @@
+# Comparison of functionality of Fluentd and Opentelemetry Collector
+
+- [Sumologic supported Fluentd plugins](#sumologic-supported-fluentd-plugins)
+  - [Sumologic Output Plugin][#sumologic-output-plugin]
+  - [fluent-plugin-datapoint][#fluent-plugin-datapoint]
+  - [fluent-plugin-protobuf][#fluent-plugin-protobuf]
+  - [fluent-plugin-prometheus-format][#fluent-plugin-prometheus-format]
+  - [fluent-plugin-kubernetes-sumologic][#fluent-plugin-kubernetes-sumologic]
+  - [fluent-plugin-kubernetes-metadata-filter][#fluent-plugin-kubernetes-metadata-filter]
+  - [fluent-plugin-enhance-k8s-metadata][#fluent-plugin-enhance-k8s-metadata]
+  - [fluent-plugin-events][#fluent-plugin-events]
```

Comment:
I believe those should be `[xxx](#anchor)`

Reply:
fixed